sormas-api/src/main/java/de/symeda/sormas/api/feature/FeatureType.java

```diff
@@ -195,7 +195,8 @@
 		true,
 		new FeatureType[] {
 			TASK_MANAGEMENT }),
-	IMMUNIZATION_MANAGEMENT(true, false, null);
+	IMMUNIZATION_MANAGEMENT(true, false, null),
```

Comment:
Please move `IMMUNIZATION_MANAGEMENT` to the "Feature Modules" section and `IMMUNIZATION_STATUS_AUTOMATION` to "Feature Extension"